scripts/init/updateCheck.sh

```diff
@@ -66,7 +59,8 @@ main() {
     image_version="$APP_VERSION"
 
     if [[ -z "$installed_version" ]]; then
-        MODULE="dbp-update" info "No installed Moodle version detected, continuing with fresh install"
+        MODULE="dbp-update" info "No installed Moodle version detected"
```

Comment:
I think one could mention that since no moodle version is detected moodle will be installed.

e.g.
```suggestion
        MODULE="dbp-update" info "No installed Moodle version detected. Installing Moodle..."
```

Reply:
Good point 👍